lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -44,9 +45,19 @@ def get_blocks(self, blocks):
         }
 
 
+class DatesWidgetSerializer(serializers.Serializer):
```

Comment:
This is just a matter of taste, but I wonder if we need a sub-serializer here. Breaking the outline API into lots of subsections with their own domain (like "tools" and "dates widget") is a natural Object Oriented impulse.

But in this case, this is special purpose API that is unlikely to grow much in complexity (the outline tab is not going to grow a wild number of features). It's possible that this level of engineering might make it harder to understand the code than its worth. (vs just throwing these 3 fields into the main `OutlineTabSerializer` serializer)

But I don't super care. Just an interesting discussion point.